Week6/app/src/main/java/com/isabelledionisius/mymovieapp/repository/MovieRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.isabelledionisius.mymovieapp.repository
+
+import androidx.lifecycle.LiveData
+import com.isabelledionisius.mymovieapp.db.DaoMovie
+import com.isabelledionisius.mymovieapp.db.MovieDataList
+
+class MovieRepository(private val movieDao: DaoMovie) {
+
+    var movieRepository =
+        MovieRepository(
+            movieDao
+        )
+
+    // Declares the DAO as a private property in the constructor. Pass in the DAO
+// instead of the whole database, because you only need access to the DAO
+    class MovieRepository(private val movieDao: DaoMovie) {
```

Comment:
I commented on this before, but I guess it didn't go through.

No need to wrap a wrapper of the DAO into another layer. Also why have two of the same class names, with the same properties?

Reply:
I have deleted the second class. However, I am not quite sure how to change that.

Reply:
Not sure what you mean. It's fine to have `class MovieRepository(private val movieDao...). But you don't need another class with the same name, doing the same thing.

Reply:
OK. Then I got you, just deleted that